app/assets/javascripts/application.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+
+document.onreadystatechange = function () {
+  var csrfElement = document.querySelector('meta[name="csrf-token"]')
+  var csrf = csrfElement ? csrfElement.content : ""
+  if (document.readyState === 'interactive') {
+    var basePath = document.getElementById('base-path-id')
+    var documentTitle = document.getElementById('document-title-id')
+    if (basePath==null || documentTitle==null) {return;}
```

Comment:
I don't particularly know our JS standards (@alex-ju do we have some?) but this feels a bit unorthadox with the lack of spacing on the nulls and the inline return